diffkemp/semdiff/result.py

```diff
@@ -84,18 +84,17 @@ def report_stat(self, show_errors=False):
             lambda x: x.diff == "", r.inner.values())) and
             r.kind == Result.Kind.NOT_EQUAL])
         if total > 0:
-            print("Total params: {}".format(total))
-            print("Equal:        {0} ({1:.0f}%)".format(eq, eq / total * 100))
-            print(" same syntax: {0}".format(eq_syn))
-            print("Not equal:    {0} ({1:.0f}%)".format(
+            print("Total symbols: {}".format(total))
+            print("Equal:         {0} ({1:.0f}%)".format(eq, eq / total * 100))
+            print(" same syntax:  {0}".format(eq_syn))
```

Comment:
Remove "same syntax" part, we're not distinguishing `equal` and `equal_syntax` at the moment.